performant-software/Annotation-Studio
# Stored all values for filterable fields into snapshot

#### What does this PR do?

Stores a slice of all annotations on a given document within the documents' snapshot; those fields from all aggregated annotations on the document which are fields on which users are likely to want to filter annotations.

Addresses #48, but the functionality won't be visible until the viewer is complete.

#### Steps to test

0. Visit: http://cove-staging-pr-78.herokuapp.com/ **[NB, not https]**
1. If you don't have an account there yet, create one, and email me so I can give you instructor permissions.
1. Annotate a document 
2. Open the tools sidebar, scroll to the bottom and click snapshot
3. Click preview.
4. View source.
5. Note that there is a script tag with the id of "filter-lists"; contained within should be three arrays of values: tags, user, and annotation_categories, containing the values you used in your annotations.
